core/themes/seven/css/seven.base.css

```diff
@@ -105,17 +115,31 @@ abbr,
 acronym {
   border-bottom: dotted 1px;
 }
-ul {
+ul,
+.block ul,
+.item-list ul{
   list-style-type: disc;
   list-style-image: none;
   margin: 0.25em 0 0.25em 1.5em; /* LTR */
+  line-height: 160%;
 }
 [dir="rtl"] ul {
   margin: 0.25em 1.5em 0.25em 0;
 }
```

Comment:
If we add `.block ul` and `.item-list ul` to the above selector, we also need to add `[dir="rtl"] .block ul` and `[dir="rtl"] .item-list ul` here to ensure RTL styles work consistently.